.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Typo ?

Reply:
To be honest this is not a typo, this is the way husky works / is setup 😄 

This `.husky/_` folder hosts the Husky code, which is installed kind of like a dependency.

How it works in a nutshell, Husky [adds a `prepare` script](https://blog.typicode.com/husky-git-hooks-autoinstall/) in the `package.json` to install Husky locally in the `.husky/_` folder. Alongside this folder, the user can add multiple git hooks files (that are versioned).

This is why we have a `.husky/.gitignore` with a `_` folder in it, to exclude the installed Husky files (located in `_`) from being versioned.

For more info, cf. the [Husky doc](https://typicode.github.io/husky/#/) and [usage](https://typicode.github.io/husky/#/?id=usage) part

Reply:
Alright then it's just a super weird and confusing choice on Husky's part 😆 
It's all good then !